src/meta/play-meta.js

```diff
@@ -113,6 +114,18 @@ export const plays = [
     cover: '',
     blog: '',
     video: ''
+  }, {
+    id: 'pl-tic-tac-toe', 
+    name: 'Tic Tac Toe', 
+    description: 'A two players Tic Tac Toe game',
+    component: () => {return <TicTacToe />},
+    path: '/plays/tic-tac-toe',
+    level: 'Beginner',
+    tags: 'useState,game',
+    github: 'joshi-kaushal',
+    cover: '',
```

Comment:
As you do not have a cover image URL, please drop a cover.png file in your play folder. Create some relevant image using tool like Canva.

Reply:
done